src/framework/musesampler/internal/musesamplerwrapper.cpp

```diff
@@ -279,7 +279,7 @@ void MuseSamplerWrapper::setIsActive(bool arg)
     m_samplerLib->setPlaying(m_sampler, arg);
 
     if (!isActive()) {
-        setCurrentPosition(m_currentPosition);
+        m_samplerLib->setPosition(m_sampler, m_currentPosition);
```

Comment:
Can you please explain why you are calling m_samplerLib->setPosition here directly?